scripts/process_vanderbilt_MxIF.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -o errexit
+
+. ./scripts/utils.sh
+
+main() {
+
+  get_CLI_args "$@"
+
+  TILES_BASE='vanderbilt_MxIF.images'
+  TILES_PATH="$OUTPUT/$TILES_BASE"
+  if [ -e "$TILES_PATH" ]
+  then
+      echo "Skipping tiling -- output already exists: $TILES_PATH"
+  else
+      mkdir $TILES_PATH
+      if [ -e "$INPUT/vanderbilt.ome.tif" ]
+      then
+        echo "Not copying $INPUT/vanderbilt.ome.tif from s3 - already exists or testing"
+      else
+        aws s3 cp s3://vitessce-data/source-data/vanderbilt_MxIF/vanderbilt.ome.tif "$INPUT/vanderbilt.ome.tif"
+      fi
+      SERVER_URL="https://vitessce-vanderbilt-data.storage.googleapis.com/"
```

Comment:
Would plain`vitessce-data` be better? ... I don't think we should but any effort into a mass migration right now, but I'm not sure having the name this specific helps us?

Reply:
Sure, I'll make a new bucket (this was my personal bucket - I forgot to push a branch before I left that has the one Nils made, `vitessce-demo-data`, but I'll change it).